quic/s2n-quic-core/src/connection/limits.rs

```diff
@@ -316,6 +319,26 @@ impl Limits {
         Ok(self)
     }
 
+    /// Sets whether the endpoint will send MTU probing complete frames (default: true)
+    ///
+    /// This parameter is only relevant when using dcQUIC endpoints. For standard QUIC
+    /// endpoints, this setting has no effect. When enabled, the endpoint will send
+    /// MtuProbingComplete frames to signal completion of MTU probing.
+    ///
+    /// Note: This is a dcQUIC-specific transport parameter and will only be transmitted
+    /// when the dcQUIC provider is enabled.
+    pub fn with_mtu_probing_complete_support(
```

Comment:
It feels a little weird for this to be in the limit configuration? I'm wondering if it should be a configuration parameter of the dcQUIC provider rather than being here?